src/Admin/OnboardingWizardSubmenu.php

```diff
@@ -39,8 +39,13 @@ public static function get_registration_action() {
 	 * Runs on instantiation.
 	 */
 	public function register() {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce verification is not needed here.
+		if ( ! isset( $_GET['page'] ) || self::SCREEN_ID !== $_GET['page'] ) {
```

Comment:
This a workaround to register the onboarding wizard submenu page under the AMP options menu page and only render it when this page is requested explicitly. 

Passing `$parent_slug` as null results in a NULL title which is now causing a deprecation warning on Onboarding Wizard pages. One more workaround is to add  `options.php` as `$parent_slug` but I think we should keep it with amp-options if the above workaround doesn't have any side effects.

Reply:
But adding `options.php` provides the same behavior without breaking anything. So updating it to `options.php`.